modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/components/StoryGradientBackground.kt

```diff
@@ -0,0 +1,44 @@
+package au.com.shiftyjelly.pocketcasts.endofyear.components
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalView
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import au.com.shiftyjelly.pocketcasts.endofyear.R
+import au.com.shiftyjelly.pocketcasts.utils.extensions.pxToDp
+
+@Composable
+fun StoryBlurredBackground(
+    offset: Offset,
+) {
+    Box(Modifier.wrapContentWidth(unbounded = true)) {
+        val context = LocalContext.current
+        val screenWidthInPx = LocalView.current.width
+        val screenWidthInDp = screenWidthInPx.pxToDp(context)
+        Box(
+            modifier = Modifier
+                .size((screenWidthInDp * 2f).dp, (screenWidthInDp * 2f).dp)
+                .offset(
+                    x = offset.x.toInt().pxToDp(context).dp,
+                    y = offset.y.toInt().pxToDp(context).dp
+                )
+        ) {
+            Image(
+                painterResource(id = R.drawable.story_blurred_background),
```

Comment:
I have used a drawable `webp` image for the blurred background. I tried using Compose's [blur](https://developer.android.com/jetpack/compose/graphics/images/customize#blur_an_image_composable) effect (available > API 31) which worked well on the phone screen, but saving it to Bitmap removed the effect and needed to be reapplied using Android's [RenderEffect](https://developer.android.com/reference/android/graphics/RenderEffect) ( > API 31) or [RenderScript](https://developer.android.com/guide/topics/renderscript/compute) which needs investigation and some extra effort.